ansible/roles/test/tasks/neighbour-mac-noptf.yml

```diff
@@ -101,18 +104,21 @@
        - ('Ethernet' in item.value['interface']) or ('PortChannel' in item.value['interface'])
        - arptable.v6 | length != 0
 
-  - name: pick Ethernet0 as test interface if cannot fine v6 neighbor to test
-    set_fact:
-      v6_intf: "Ethernet0"
-      v6_nei: "{{ v6_intf_nei }}"
-    when: v6_nei is not defined
+  - block:
+    - name: pick Ethernet0 as test interface if cannot fine v6 neighbor to test
+      set_fact:
+        v6_intf: "Ethernet0"
+        v6_nei: "{{ v6_intf_nei }}"
 
-  - name: add an ipv6 entry for Ethernet0 if not find v6 neighbor
-    command: "/sbin/ifconfig Ethernet0 inet6 add {{ v6_intf_ip }}"
-    when: v6_nei is not defined
+    - name: startup Ethernet0
+      command: "config interface startup Ethernet0"
+
+    - name: add an ipv6 entry for Ethernet0 if not find v6 neighbor
```

Comment:
what do you mean if not found? you are adding it.
if you keep this sentence suggest to rephrase "add an ipv6 entry for Ethernet0 if v6 neighbor was not found"

Reply:
fixed.